README.md

```diff
@@ -21,6 +24,38 @@ Barnabas provides a way to run an [Apache Kafka][kafka] cluster on
 
 <!-- /TOC -->
 
+## Deploying a Kubernetes/OpenShift cluster
+
+You will need a Kubernetes or OpenShift cluster to deploy Barnabas.
+
+### Kubernetes
+
+In order to interact with a Kubernetes cluster, be sure to have [the kubectl tool][kubectl] installed.
+If you don't have a Kubernetes cluster up and running, the easiest way to deploy one, for development purposes, is to use the [Minikube][minikube] project which provides a single-node cluster in a VM. In order to do that you can just follow the installation guide which describes all the prerequisites and how to get the Minikube binaries.
+Finally, the cluster can be deployed by running
+
+```
+minikube start
+```
+
+### OpenShift
+
+In order to interact with an OpenShift cluster, be sure to have the [Openshift client tools][origin] installed.
```

Comment:
It should be `[OpenShift client tools][origin]` (if we're consistently camelcasing it)